Extending/Macro-Parameter-Editors/index-v8.md

```diff
@@ -25,7 +25,7 @@ It is possible to create custom macro parameter types.
 
 ### isParameterEditor
 
-To create a custom Macro Parameter Type, first create a custom 'Property Editor' (or copy one from the core). See [Property Editors documentation](../../Extending/Property-Editors/index-v8.md) and in the corresponding [Package Manifest file](../../Extending/Property-Editors/Package-Manifest/index.md) for the editor, set the `isParameterEditor` property to be true.
+To create a custom Macro Parameter Type, first create a custom 'Property Editor' (or copy one from the core). See [Property Editors documentation](../../Extending/Property-Editors/) and in the corresponding [Package Manifest file](../../Extending/Property-Editors/Package-Manifest/index.md) for the editor, set the `isParameterEditor` property to be true.
```

Comment:
Shouldn't this instead link to the [Umbraco 7-8 version of the "Property Editor documentation"](https://our.umbraco.com/Documentation/Extending/Property-Editors/index-v7-8)?
Since this article is marked for Umbraco 8, I think it should also link to Umbraco 8 valid docs, right? 🤔